src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const ul = document.querySelector('ul');
+const img = document.querySelector('#largeImg');
+const eventName = 'click';
+
+const func = (action) => {
```

Comment:
A bad synonym for the word `event`. `Action` - means that there is a method in the variable, and you actually have a DOM element. Rename it please.

Reply:
Actually, I want to use event, but linter rejected this word